.nais/deploy-prod.yml

```diff
@@ -106,6 +106,10 @@ spec:
       value: prod-fss:pensjonsamhandling:tp
     - name: TJENESTEPENSJON_URL
       value: https://tp-api.prod-fss-pub.nais.io
+    - name: OMSTILLINGSSTOENAD_SERVICE_ID
+      value: prod-gcp:etterlatte:etterlatte-samordning-vedtak
+    - name: OMSTILLINGSSTOENAD_SERVICE_URL
+      value: https://etterlatte-samordning-vedtak.intern.nav.no
```

Comment:
Mangler det under `outbound` `rules` for denne?

Reply:
Det gjør de, ja 👍